docs/tools/developer-tools/lab.mdx

```diff
@@ -2,7 +2,7 @@
 title: Lab
 description: Explore Lab, a developer tool to learn about the Stellar network
 sidebar_label: Lab
-sidebar_position: 70
```

Comment:
Alternatively, we could probably remove the `sidebar_position` for all of the pages after `data indexers` because it seems like Docusaurus will put them in alphabetical order anyway. That will be less to update in the future, but the way it currently is, is a bit more explicit.

Reply:
I think what you have is good for sidebar positioning.